### PR TITLE
Add proto-steering group minutes folder and first set of minutes

### LINKED ISSUE #152: Add Steering Group minutes folder and backlog of minutes
Prior to official initiation of the ESMValTool consortium and Steering Group, we are operating interim group to develop strategy and coordinate proposals: the _proto-Steering Group_. This group has held monthly meetings since November 2023, and have agreed that notes from these meetings should be published in the Community repository.

### PR DESCRIPTION
**Pull request by alistairsellar**
Add minutes of proto-Steering group meetings to the community repository.

This group has superseded the _cross-team coordination group_ so it would make sense to make the `Cross team` folder less prominent, and also to future-proof for the proto-Steering Group being superseded by the actual Steering Group. I propose to group them all under a `Steering Group` folder as follows:
```
|-- Steering\ Group
|   |-- Minutes
|   |-- README.md
|   |-- Cross\ Team
|   |   |-- Minutes
|   |   `-- README.md
|   `-- proto-Steering\ Group
|       |-- Minutes
|       `-- README.md
```
The top-level `README.md` of the community repository should link to the proto-Steering Group `README.md` for now, and it will be updated again to link to the actual Steering Group when that is initiated.

Closes #152

### CODE REVIEW

**Comment by alistairsellar** [comment]
Hi @hb326 @bouweandela @axel-lauer @rswamina @sloosvel please could one of you review this PR to add the proto-Steering Group notes to the community repo? I hope that the proposed directory structure is clear?

File: Steering Group/proto-Steering Group/Minutes/20240116.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+
+## Community meetings
+
+Birgit & Bouwe met with Forrest Hoffman (ORNL) & Carlo Buentempo (Copernicus Climate Change Service) and others about the proposal for a shared fixes package. Carlo is keen to engage with this effort and there is potential for some funding.
+
+Related discussion on _reducing_ errors in the first place. Potential for ESGF developments to help modelling centres publish fewer metadata errors, by improving checks performed in the publication process. We should aim to tackle both ends, reduce errors but accept that there will always be some errors that need fixing.
+
+Another meeting with this group is scheduled for February.
+**ACTION**: Birgit to propose to Forrest that Ag and Carston are invited to the next fixes meeting with Carlo, et al.
```
Comment (bouweandela) [inline Steering Group/proto-Steering Group/Minutes/20240116.md:58]:
```suggestion
**ACTION**: Birgit to propose to Forrest that Ag and Carsten are invited to the next fixes meeting with Carlo, et al.
```

File: Steering Group/proto-Steering Group/Minutes/20240116.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+1. Managing connections and dealing with knowledge gaps when personnel move: 
+    a. Sub-teams need to discuss and plan towards this. We also need to build capacity.
+    b. Bouwe to invite partners from ACCESS-NRI to Tech Team meetings.
+    
+2. ESMValTool Strategy document
+    b. Everyone to provide input to the documents as well.
+        
+3. Obs data versioning: Technical team to discuss [GitHub issue #3452](https://github.com/ESMValGroup/ESMValTool/issues/3452)
+
+4. Birgit to propose to Forrest that Ag and Carston are invited to the next fixes meeting with Carlo, et al.
```
Comment (bouweandela) [inline Steering Group/proto-Steering Group/Minutes/20240116.md:95]:
```suggestion
4. Birgit to propose to Forrest that Ag and Carsten are invited to the next fixes meeting with Carlo, et al.
```

**Review by bouweandela — APPROVED** [review]
Thanks @alistairsellar!

**Comment by alistairsellar** [comment]
Thanks @bouweandela!

**Pull request merged**